src/Observers/UserObserver.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Observers;
+
+use Blumilk\Meetup\Core\Models\User;
+use Illuminate\Contracts\Hashing\Hasher;
+
+class UserObserver
+{
+    public function __construct(
+        public Hasher $hasher,
```

Comment:
```suggestion
        protected Hasher $hasher,
```

It should not be public in any way.